docs/UserManuals/ConfigUI/GitHub.md

```diff
@@ -23,6 +23,13 @@ GitHub personal access tokens are required to add a connection.
   - `read:org`
 - The data collection speed is relatively slow for GitHub since they have a **rate limit of [5,000 requests](https://docs.github.com/en/rest/overview/resources-in-the-rest-api#rate-limiting) per hour** (15,000 requests/hour if you pay for GitHub enterprise). You can accelerate the process by configuring _multiple_ personal access tokens. Please note that multiple tokens should be created by different GitHub accounts. Tokens belonging to the same GitHub account share the rate limit.
 
+#### Use Graphql APIs
+
+If you use `github.com` or your private GitHub Service supported, you could open this switcher to collect data more swiftly.
+
+- GraphQL APIs are 10+ times faster than REST APIs, but it may not be supported in GitHub on-premise versions. 
```

Comment:
Text change:
"GraphQL APIs are 10+ times faster than REST APIs, but they may not be supported in GitHub on-premise versions."